multiarch/generate-pipeline.groovy

```diff
@@ -53,21 +53,13 @@ node {
 		def archImages = [:]
 
 		// https://github.com/docker-library/meta/blob/subset/subset.txt
-		def subset = [
-			'cassandra',
-			'docker',
-			'drupal',
-			'gcc',
-			'ghost',
-			'julia',
-			'memcached',
-			'mysql',
-			'notary',
-			'postgres',
-			'rabbitmq',
-			'redis',
-			'redmine'
-		]
+		def subset = []
+		new URL('https://raw.githubusercontent.com/docker-library/meta/subset/subset.txt').eachLine { line ->
+			subset << line
```

Comment:
Does it make a difference that in the txt file the names are quoted but not here?

Reply:
😕 The [subset.txt](https://github.com/docker-library/meta/blob/subset/subset.txt) file isn't quoted, just one image per line. The quoting was in this file to fit "strings in an array" syntax.

Reply:
Ah yeah, I mean the other way around